capstone-frontend/src/BookSearchTile.test.js

```diff
@@ -6,28 +6,55 @@ import { act } from "react-dom/test-utils";
 import BookSearchTile from "./components/BookSearchTile.jsx";
 import App from "./App.jsx";
 import {
-  Route, 
-  BrowserRouter as Router, 
+  Route,
+  BrowserRouter as Router,
   Link
-} from 'react-router-dom'; 
+} from 'react-router-dom';
 
 const container = document.createElement("div");
 document.body.appendChild(container);
 const book = {
   title: "Title",
-  author: ["Author"],
+  authors: ["Author"],
   thumbnailLink: "https://http.cat/100.jpg"
 };
+const getUserBookLists = () => {
+  let userBookLists = [{ id: "1", name: "Best Books" },
+  { id: "2", name: "Fantasy" },
+  { id: "3", name: "Sci-fi" }];
+  return userBookLists;
```

Comment:
nit: This would probably be a bit cleaner if it just directly returned the array literal instead of first storing it in a variable. If you decide to keep the variable, use `const` since it isn't reassigned.

Same applies to the helper functions below.

Reply:
That's a good idea! I've fixed that now